pkg/config/config.go

```diff
@@ -24,8 +24,9 @@ import (
 
 // Finch represents the configuration file for Finch CLI.
 type Finch struct {
-	CPUs   *int    `yaml:"cpus"`
-	Memory *string `yaml:"memory"`
+	CPUs                  *int     `yaml:"cpus"`
+	Memory                *string  `yaml:"memory"`
+	AdditionalDirectories []string `yaml:"additional_directories,omitempty"`
```

Comment:
Might be better to make this a slice of a struct, even if the struct just has one entry, like path, that way its non-breaking to add more fields to the struct in the future
```golang
type AdditionalDirectory struct {
    Path string
}
```

```suggestion
	AdditionalDirectories []AdditionalDirectory `yaml:"additional_directories,omitempty"`
```